application/apps/precompiled/updater/src/env/args.rs

```diff
@@ -20,7 +20,10 @@ pub struct Arguments {
     pub app_name: String,
     pub location: PathBuf,
     pub compressed: PathBuf,
+    // `pid` and `ppid` aren't used yet but they can be used to check shutdown status by parent process.
+    #[allow(dead_code)]
     pub pid: Option<usize>,
+    #[allow(dead_code)]
```

Comment:
@marcmo I believe that's okay to ignore these warns in this case. That's actually not an implemented feature of the updater, which should check the state of the parent process and start updating as soon as the process is down. As soon as we didn't find quick solution for Windows we postpone this feature. But the parent process sends to updater PID as well and it makes sense to leave it there.